ironfish/src/network/peers/peerManager.ts

```diff
@@ -1445,10 +1444,6 @@ export class PeerManager {
         continue
       }
 
-      if (connectedPeer.port !== null && connectedPeer.address === null) {
```

Comment:
Changing handling of PeerList messages here. If the peer candidate does not contain an `address` then ignore the port as well